contracts/src/FeeContract.sol

```diff
@@ -13,18 +13,17 @@ contract FeeContract is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     //
     /// @notice Notify a new deposit
     event Deposit(address indexed user, uint256 amount);
-    event Log(string func, uint256 gas);
 
     /// @notice upgrade event when the proxy updates the implementation it's pointing to
     event Upgrade(address implementation);
+
     // === Constants ===
     //
     /// @notice max amount allowed to be deposited to prevent fat finger errors
     // @TODO confirm this amount with product
+    uint256 public maxDepositAmount = 1 ether;
```

Comment:
as it's an upgradable contract - we can't assign values to state variables on the implementation contract since we actually use the storage on the proxy contract. Only set these values in the `initialize` function